topics/admin/tutorials/connect-to-compute-cluster/slurm-wlm-configurator.html

```diff
@@ -0,0 +1,842 @@
+If Sockets is omitted, it will be inferred from:
+CPUs, CoresPerSocket, and ThreadsPerCore.
+<P>
+<input type="text" name="cores_per_socket" value="">
+<B>CoresPerSocket</B>:
+Number of cores in a single physical processor socket.
+The CoresPerSocket value describes physical cores, not
+the logical number of processors per socket.
+<P>
+<input type="text" name="threads_per_core" value="">
+<B>ThreadsPerCore</B>:
+Number of logical threads in a single physical core.
+<P>
+<input type="text" name="memory" value=""> <B>RealMemory</B>: Amount
+of real memory. This parameter is required when specifying Memory as a
+consumable resource with the select/cons_res plug-in. See below
+under Resource Selection.
+<P>
+
+<H2>SLURM User</H2>
+The SLURM controller (slurmctld) can run without elevated privileges,
+so it is recommended that a user "slurm" be created for it. For testing
+purposes any user name can be used.
+<P>
+<input type="text" name="slurm_user" value="slurm"> <B><font color="red">SlurmUser</font></B>
+<P>
+
+<H2>Group ID Caching</H2>
+
+If you have a slow NIS environment, big parallel jobs take a long time
+to start up (and may eventually time-out) because the NIS server(s)
+may not be able to quickly respond to simultaneous requests from
+multiple slurmd's.  You can instruct slurmd to cache /etc/groups
+entries to prevent this from happening by setting
+<B>CacheGroups</B>=1.  Reconfiguring ("scontrol reconfig") with
+<B>CacheGroups</B>=0 will cause slurmd to purge the cache.  Select one
+value for <B>CacheGroups</B>:<BR>
+<input type="radio" name="cache_groups" value="0" checked>
+<B>0</B>: for normal environment.<BR>
+<input type="radio" name="cache_groups" value="1">
+<B>1</B>: for slow NIS environment.
+<P>
+WARNING: The group ID cache does not try to keep itself in sync with
+the system.  You MUST run "scontrol reconfig" to update the cache
+after making any changes to system password or group databases.
+<P>
+
+<H2>SLURM Port Numbers</H2>
+The SLURM controller (slurmctld) requires a unique port for communications
+as do the SLURM compute node daemons (slurmd). If not set, slurm ports
+are set by checking for an entry in <I>/etc/services</I> and if that
+fails by using an interval default set at SLURM build time.
+<P>
+<input type="text" name="slurmctld_port" value="6817"> <B>SlurmctldPort</B>
+<P>
+<input type="text" name="slurmd_port" value="6818"> <B>SlurmdPort</B>
+<P>
+
+<H2>Authentication and Security</H2>
+Define the method used for authenticating communicating between SLURM components.<BR>
+Select one value for <B>AuthType</B>:<BR>
+<input type="radio" name="auth_type" value="none"> <B>None</B>: No authentication,
+not recommended production use<br>
+<input type="radio" name="auth_type" value="authd"> <B>Authd</B>: Brent Chun's
+<A href="http://www.theether.org/authd/">authd</A><BR>
+<input type="radio" name="auth_type" value="munge" checked> <B>Munge</B>: LLNL's
+<A href="http://home.gna.org/munge/">Munge</A><BR>
```

Comment:
This link is failing (time out)

Reply:
It seems to be the correct URL. I can change it to https://github.com/dun/munge in the meantime.